app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt

```diff
@@ -186,4 +193,11 @@ class ConversationsFragment : SFragment(), StatusActionListener, Injectable {
     companion object {
         fun newInstance() = ConversationsFragment()
     }
+
+    private val refreshReceiver = object : BroadcastReceiver() {
+        override fun onReceive(context: Context, intent: Intent) {
+            //refresh
+            initSwipeToRefresh()
```

Comment:
That does not look right, I think you just have to call `viewModel.refresh()` here